google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -29,15 +29,27 @@ class Table:
     Attributes:
         documentai_table (google.cloud.documentai.Document.Page.Table):
             Required. The original google.cloud.documentai.Document.Page.Table object.
+        document_text (str):
+            Required. UTF-8 encoded text in reading order from the document.
```

Comment:
Is `document_text` the text for the whole document, or just that of a page containing the object (e.g. a Table)?  If it is just the containing page's text, perhaps let's replace the `document_text` attribute with a `_page` attribute instead (here and below).

Reply:
Throughout this code, `document_text` is the text of the full documentai Document, not just the page.